zhmccli/_cmd_partition.py

```diff
@@ -916,6 +967,133 @@ def cmd_partition_console(cmd_ctx, cpc_name, partition_name, options):
             "{exc}: {msg}".format(exc=exc.__class__.__name__, msg=exc))
 
 
+def cmd_partition_dump(cmd_ctx, cpc_name, partition_name, **options):
+    # pylint: disable=missing-function-docstring
+
+    client = zhmcclient.Client(cmd_ctx.session)
+    partition = find_partition(cmd_ctx, client, cpc_name, partition_name)
+
+    name_map = {
+        # These parameters are the same for CPCs with and without the storage
+        # management feature. Feature-specific parameters are added further
+        # down:
+        'configuration': 'dump-configuration-selector',
+        'parameters': 'dump-os-specific-parameters',
+        'lba': 'dump-record-lba',
+
+        # Options that are handled in this function:
+        'volume': None,
+        'timeout': None,
+    }
+    options = original_options(options)
+    properties = options_to_properties(options, name_map)
+    volume_opt = options['volume']  # It is required
+    timeout = options['timeout']  # It is optional but has a default
+
+    if storage_management_feature(partition):
+
+        # Check and parse the --volume option into SG.SV or UUID
+        volume_parts = volume_opt.split('.')
+        if len(volume_parts) == 1:  # format: UUID
+            uuid = volume_parts[0]
```

Comment:
```suggestion
            uuid, = volume_parts
```

Reply:
I don't understand the motivation for this change. Besides that, the new code does not work on Python 2.7:
```
Python 2.7.16 (default, Sep  2 2019, 11:59:44) 
[GCC 4.2.1 Compatible Apple LLVM 10.0.1 (clang-1001.0.46.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> a=(1,2,3)
>>> b, = a
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: too many values to unpack
>>> 
```
REJECTED.

Reply:
The intention behind this was that it would make refactoring the code easier. But right, it doesn't work for Python2.